jenkins/requirements.txt

```diff
@@ -1 +0,0 @@
-requests==2.5.3
```

Comment:
Oh at this point, and with the general simplicity of the scripts in these directories, I also think one test-requirements or requirements file makes a lot of sense! But, I am just not sure if there are scripts which infer that you install requirements from the jenkins folder.

If you just spend a few minutes to grok for any documentation on those, that'd be great. I will have to check job config on jenkins to see how we pull in the requirements.txt files...